ktlint-custom/src/main/kotlin/com/baeldung/ktlintcustomrule/TestNamingRule.kt

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.ktlintcustomrule
+
+import com.pinterest.ktlint.rule.engine.core.api.ElementType
+import com.pinterest.ktlint.rule.engine.core.api.Rule
+import com.pinterest.ktlint.rule.engine.core.api.RuleId
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+class TestNamingRule :
+    Rule(
+        ruleId = RuleId("$CUSTOM_RULE_SET_ID:unit-test-convention"),
```

Comment:
about = About(maintainer = "Gaetano Piazzolla"),
  --> could we remove this section, or is it required?